src/virtualship/cli/commands.py

```diff
@@ -45,9 +47,74 @@ def init(path):
     "path",
     type=click.Path(exists=True, file_okay=False, dir_okay=True, readable=True),
 )
-def fetch(path):
-    """Entrypoint for the tool."""
-    raise NotImplementedError("Not implemented yet.")
+def fetch(expedition_dir: str | Path) -> None:
+    """Entrypoint for the tool to download data based on area of interest."""
```

Comment:
its here, click expects the parameter name to be `path` to match the decorator

Reply:
OK, so am I doing something wrong then when I do `virtualship fetch TrAtldrifters` (and `TrAtldrifters` is the name of a directory that exists)?

Reply:
No, its an error in the code

```suggestion
def fetch(path: str | Path) -> None:
    """Entrypoint for the tool to download data based on area of interest."""
```
+ an update to the other references. Should I push a patch for this? (+ my other review comments)

Reply:
yes please!

Reply:
Resolved now